druid/src/event.rs

```diff
@@ -397,8 +397,22 @@ impl Event {
         }
     }
 
-    /// Whether this event should be sent to widgets which are currently not visible
-    /// (for example the hidden tabs in a tabs widget).
+    /// Whether this event should be sent to widgets which are currently not visible and not
+    /// accessible.
+    /// Therefore the hidden tabs in a tabs widget are `hidden` whereas the hidden
+    /// widgets in a scroll are not, since you can bring them into view by scrolling.
+    ///
+    /// This distinction between scroll and tabs is due to one of the main effects of
+    /// this method: determine which widgets are allowed to receive focus. As a rule
+    /// of thumb a widget count as `hidden` if it makes no sense for it to receive focus
+    /// via pressing TAB.
+    ///
+    /// If a widget changes which children are hidden it must call [`children_changed`].
+    ///
+    /// See also [`LifeCycle::should_propagate_to_hidden`]
+    ///
+    /// [`children_changed`]: crate::EventCtx::children_changed
+    /// [`LifeCycle::should_propagate_to_hidden`]: LifeCycle::should_propagate_to_hidden
```

Comment:
```suggestion
    /// Whether this event should be sent to widgets which are currently not visible and not
    /// accessible.
    ///
    /// For example: the hidden tabs in a tabs widget are `hidden` whereas the non-visible
    /// widgets in a scroll are not, since you can bring them into view by scrolling.
    ///
    /// This distinction between scroll and tabs is due to one of the main purposes of
    /// this method: determining which widgets are allowed to receive focus. As a rule
    /// of thumb a widget counts as `hidden` if it makes no sense for it to receive focus
    /// when the user presses thee 'tab' key.
    ///
    /// If a widget changes which children are hidden it must call [`children_changed`].
    ///
    /// See also [`LifeCycle::should_propagate_to_hidden`].
    ///
    /// [`children_changed`]: crate::EventCtx::children_changed
    /// [`LifeCycle::should_propagate_to_hidden`]: LifeCycle::should_propagate_to_hidden
```